src/Hooks.php

```diff
@@ -81,6 +81,13 @@ public static function onSpecialSearchSetupEngine( $search, $profile, $engine )
 	 * @return string
 	 */
 	public static function addMatomo( $title ) {
+		
+		// Fix #44. This function is being called twice from OutputPage (seems to be an upstream bug).
+		static $called = false;
+
+		if($called) { return; }
```

Comment:
How will `$called` ever be anything other than `false` at this point? It looks like this will never return early.

Reply:
In PHP a static variable defined in a function only exists within that function (or method in this case) but does not lose its value upon the function returning. See https://www.php.net/manual/en/language.variables.scope.php#language.variables.scope.static